HACKING.md

```diff
@@ -38,7 +38,7 @@ When submitting your pull request, please make sure you:
 
 * Add tests.
   C functions should have unit tests.
-  Promise types, attributes and functions should have acceptance tests.
+  Promise types, attributes, and functions should have acceptance tests.
```

Comment:
Why this comma? (I'm no expert on English comma-grammar, but it feels weird here)